docs/src/ff_embedding.md

```diff
@@ -8,36 +8,52 @@ CurrentModule = Nemo
 
 Nemo allows the construction of finite field embeddings making use of the
 algorithm of Bosma, Cannon and Steel behind the scenes to ensure compatibility.
+Critical routines (e.g. polynomial factorization, matrix computations) are
+provided by the C library Flint, whereas high level tasks are written directly in Nemo.
 
 ## Embedding functionality
 
-### First set of functions
+It is possible to explicitly call the embedding `embed` function to create an embedding,
+but standard object oriented conversion works too. The resulting embedding is of
```

Comment:
"object oriented conversion" - a bit too technical for a mainly mathematical audience. Could you use a few words to explain what you mean here.

Reply:
I do give an example in the julia code below. Do you want to have an other one in "Embedding functionality"? Or should I juste remove from the julia code?

Reply:
I just meant say what you mean by "a field". Do you mean, "a finite field" or something like e.g, the rationals.